pipeline.yml

```diff
@@ -39,20 +39,13 @@ resources:
     repository: om
     access_token: {{GITHUB_TOKEN}}
 
-- name: tool-om-beta
-  type: github-release
-  source:
-    user: c0-ops
-    repository: om
-    access_token: {{GITHUB_TOKEN}}
-
 - name: pivnet-elastic-runtime
   type: pivnet
   check_every: 4h
   source:
     api_token: {{PIVNET_TOKEN}}
     product_slug: elastic-runtime
-    product_version: 1\.9\..*
+    product_version: 1\.10\..*
```

Comment:
@jtgammon did you bump the Ops Mgr version to 1.10 also? It looks like here you just bumped ERT to 1.10